code/src/index.css

```diff
@@ -5,9 +5,143 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
-}
+  background-color: black;
```

Comment:
It is more about personal preferences to have it black or not. It would be easier to see the album cards with the gray color instead.